akka-docs-new/src/main/paradox/guide/quickstart.md

```diff
@@ -62,52 +62,61 @@ organization := "intro-akka.organization"
 version := "0.1-SNAPSHOT"
 
 scalaVersion := "2.11.8"
-val AkkaVersion = "2.4.12"
+val AkkaVersion = "2.4.17"
```

Comment:
Oh I think we may not yet do it in the new docs... It would have to be set in `akka-docs-new/build.sbt` like so:


```
paradoxProperties ++= Map(
  "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
  "akka.version" -> version.value // I guess like this?
)

```